server/src/main/java/org/apache/calcite/avatica/server/PropertyBasedSpnegoLoginService.java

```diff
@@ -60,7 +61,8 @@ public PropertyBasedSpnegoLoginService(String realm, String serverPrincipal) {
     targetNameField.set(this, serverPrincipal);
   }
 
-  @Override public UserIdentity login(String username, Object credentials) {
+  @Override public UserIdentity login(String username, Object credentials,
+      ServletRequest request) {
```

Comment:
From the looks of https://github.com/eclipse/jetty.project/commit/c23f21c761ff56f2ad4841a98083b237494442de, the method signature changed, but there were no changes of consequence in SpnegoLoginService. We're good.